test/resources/layout-shared.css

```diff
@@ -26,3 +26,35 @@ body {
   padding: 10px;
   font-size: 14px;
 }
+
+.tile-playground {
+    padding: 10px;
+    background: white;
+    font-size: 14px;
+}
+
+.product-tile-playground {
+  padding: 10px;
+  background: white;
+  font-size: 14px;
+  width: 25%;
+}
+
+.media-tile-playground {
+  display: table-cell;
+  padding: 5px;
+  vertical-align: middle;
+}
+
+.fd-has-grid-row-span-2 {
+  grid-row: span 2;
+} 
+
+.fd-has-grid-column-span-2 {
+  grid-column: span 2;
+}
+
+.fd-has-background-color-accent-7 {
+  background-color: var(--fd-color-accent-7, #0cd7ed) !important;
+}
+
```

Comment:
^^ Not sure if this is the best place for the fd-has-gride-row-span-2, fd-has-grid-column-span-2, and fd-has-background-color-accent-7 helper functions.